test/parallel/test-http-client-aborted-event.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const common = require('../common');
+const http = require('http');
+
+const server = http.Server(function(req, res) {
+  console.log('Server accepted request.');
```

Comment:
Oh, wait, we definitely directly output TAP sometimes from JS (e.g., when a test is skipped), so yeah, I guess it is A Thing...